.travis.yml

```diff
@@ -0,0 +1,18 @@
+install: gradle wrapper --gradle-version 4.2
```

Comment:
This seems to be required to run the TravisCI build. I think it's also possible to install the jar in the repo and commit that, but I think this might be cleaner